### ISSUE TITLE
Event time labels hidden when overlap

### ISSUE THREAD

**Issue by stevensturkop** [jquense/react-big-calendar#1839]
#### Do you want to request a _feature_ or report a _bug_?
Report a bug

#### What's the current behavior?
When multiple events are overlapping, some are being hidden at times

![image](https://user-images.githubusercontent.com/42664988/103462561-58de6d00-4d26-11eb-8002-b08cd92545b1.png)
![image](https://user-images.githubusercontent.com/42664988/103463412-7b738480-4d2c-11eb-97dd-5ffe6e067116.png)

These are extreme cases, nevertheless, they align perfectly when using fullcalendar.

#### What's the expected behavior?
The hidden event should be on top to display the time labels OR have a smaller left offset OR both. If anyone is keen to help fix that :)

In case someone wants to test, here is a test array:

`
[
    {
      start:
        "Sun Jan 03 2021 10:30:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 14:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 10:30:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 12:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 11:00:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 14:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 11:30:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 14:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 11:30:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 12:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 12:30:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 13:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 10:30:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 12:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 11:00:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 14:30:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 12:00:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 13:00:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 13:00:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 17:00:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 14:00:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 15:00:00 GMT+0100 (Central European Standard Time)",
    },
    {
      start:
        "Sun Jan 03 2021 14:00:00 GMT+0100 (Central European Standard Time)",
      end: "Sun Jan 03 2021 23:00:00 GMT+0100 (Central European Standard Time)",
    },
  ]
`

**Comment by ckcks12** [comment]
TL; DR
```
dayLayoutAlgorithm="no-overlap" // overlap -> no-overlap
```

https://github.com/jquense/react-big-calendar/issues/1397#issuecomment-755283689

Hope that people having this same issue can notice this attribute.
Which is under the hood for now 😅

**Comment by stiveurkel** [comment]
@ckcks12 I saw it, but the point for me would be to have the overlapping visual effect. And I can see that you are handling both:
- overlap.js
- no-overlap.js

So the overlap.js fails in some edge cases.

**Comment by ckcks12** [comment]
overlap is kinda "no algorithm".
that file is just a copy of basic algorithm.
